elvis.config

```diff
@@ -6,16 +6,54 @@
     [#{dirs => ["src"],
        filter => "*.erl",
        ruleset => erl_files,
-       rules => [{elvis_style, line_length, #{limit => 100,
-                                              skip_comments => false}},
-		 {elvis_style, dont_repeat_yourself, #{min_complexity => 20}}
-		]
+       rules => [{elvis_text_style, line_length, #{limit => 100,
+                                                   skip_comments => false}},
+		             {elvis_style, dont_repeat_yourself, #{min_complexity => 20}}
+                ],
+       ignore => ['XmppAddr', acl, adhoc, amp, amp_resolver, amp_strategy, cert_utils, ejabberd,
+                  ejabberd_admin, ejabberd_app, ejabberd_cowboy, ejabberd_ctl, ejabberd_local,
+                  ejabberd_router, ejabberd_s2s, ejabberd_s2s_in, ejabberd_s2s_out,
+                  ejabberd_service, ejabberd_sm, ejabberd_sm_cets, ejabberd_sm_mnesia,
+                  ejabberd_sm_redis, ejabberd_sup, eldap_filter, eldap_filter_yecc, eldap_pool,
+                  eldap_utils, gdpr_api, gen_hook, gen_iq_handler, gen_mod, gen_mod_deps, jlib,
+                  just_tls, mnesia_api, mod_adhoc, mod_amp, mod_auth_token_rdbms, mod_bind2,
+                  mod_bosh, mod_bosh_socket, mod_cache_users, mod_caps, mod_caps_mnesia,
+                  mod_carboncopy, mod_csi, mod_disco, mod_domain_isolation, mod_extdisco,
+                  mongoose_data_forms, mongoose_disco, mod_last_mnesia, mod_last_rdbms, mod_ping,
+                  mod_presence, mod_private, mod_private_mnesia, mod_private_rdbms,
+                  mod_push_service_mongoosepush, mod_register, mod_sasl2, mod_sic, mod_time,
+                  mod_websockets, mongoose_acc, mongoose_account_api, mongoose_amqp,
+                  mongoose_api_common, mongoose_bin, mongoose_cets_discovery,
+                  mongoose_cets_discovery_rdbms, mongoose_cleaner, mongoose_cluster,
+                  mongoose_deprecations, mongoose_http_client, mongoose_instrument_probe_cets,
+                  mongoose_iq_worker, mongoose_lib, mongoose_local_delivery, mongoose_mnesia,
+                  mongoose_packet_handler, mongoose_privacy, mongoose_rabbit_worker, mongoose_redis,
+                  mongoose_router_external, mongoose_router_external_localnode,
+                  mongoose_router_global, mongoose_router_localdomain, mongoose_scram,
+                  mongoose_server_api, mongoose_session_api, mongoose_start_node_id, mongoose_tls,
+                  mongoose_transport, mongoose_user_cache, mongoose_xmpp_errors, monitored_map,
+                  translate, tr_util, mod_last]
       },
      #{dirs => ["test", "test.disabled/mongoose_tests/tests"],
        filter => "*.erl",
        ruleset => erl_files,
-       rules => [{elvis_style, line_length, #{limit => 100,
-                                              skip_comments => false}},
+       ignore => [amp_SUITE, amp_gen, auth_dummy_SUITE, auth_http_SUITE, auth_internal_SUITE,
```

Comment:
The order of options here is different from the section above (ruleset, rules, ignore vs. ruleset, ignore, rules). Could you adjust one of them so they match?